packages/compat/src/prepare-htmlbars-ast-plugins.ts

```diff
@@ -1,32 +1,7 @@
-import { join } from 'path';
+import type { Transform } from 'babel-plugin-ember-template-compilation';
 
-export default function prepHtmlbarsAstPluginsForUnwrap(registry: any): void {
-  for (let wrapper of registry.load('htmlbars-ast-plugin')) {
-    const { plugin, parallelBabel, baseDir, cacheKey } = wrapper;
-    if (plugin) {
-      // if the parallelBabel options were set on the wrapper, but not on the plugin, add it
-      if (parallelBabel && !plugin.parallelBabel) {
-        plugin.parallelBabel = {
-          requireFile: join(__dirname, 'htmlbars-unwrapper.js'),
-          buildUsing: 'unwrapPlugin',
-          params: parallelBabel,
-        };
-      }
-
-      // NOTE: `_parallelBabel` (not `parallelBabel`) is expected by broccoli-babel-transpiler
-      if (plugin.parallelBabel && !plugin._parallelBabel) {
-        plugin._parallelBabel = plugin.parallelBabel;
-      }
-
-      // if the baseDir is set on the wrapper, but not on the plugin, add it
-      if (baseDir && !plugin.baseDir) {
-        plugin.baseDir = baseDir;
-      }
-
-      // if the cacheKey is set on the wrapper, but not on the plugin, add it
-      if (cacheKey && !plugin.cacheKey) {
-        plugin.cacheKey = cacheKey;
-      }
-    }
-  }
+export default function loadAstPlugins(registry: any): Transform[] {
+  let plugins = registry.load('htmlbars-ast-plugin').map((wrapper: any) => wrapper.plugin);
+  plugins.reverse();
```

Comment:
why's the reverse needed?

Reply:
The plugins in the registry have historically always run in the backwards order for dumb reasons. *sigh*.

I should make sure the comment doesn't get dropped by this PR though, if you look through the diff you can see the spot where it's getting removed.